mods/ra/sequences/structures.yaml

```diff
@@ -598,6 +598,40 @@ brik:
 		Length: 16
 	icon: brikicon
 
+vgate:
+	make:
+		Start: 0
+		Length: 7
+	idle:
+		Frames: 6, 5, 4, 3, 2, 1, 0
+		Length: 7
+		ZOffset: -1c511
+	damaged-idle:
+		Frames: 13, 12, 11, 10, 9, 8, 7
+		Length: 7
+		ZOffset: -1c511
+
+agate:
+	make:
+		Start: 0
+		Length: 7
+	idle:
+		Frames: 6, 5, 4, 3, 2, 1, 0
+		Length: 7
+		ZOffset: -1c511
+	damaged-idle:
+		Frames: 13, 12, 11, 10, 9, 8, 7
+		Length: 7
+		ZOffset: -1c511
+
+sgate:
+	make:
+		Start: 0
+		Length: 6
+	idle:
+		Start: 6
+		Length: -6
+
```

Comment:
The whole sgate sequence definition looks quite unfinished (Length is wrong, no damage stage, no lower ZOffset), I recommend simply using `Inherits: agate`.